redis/src/cluster.rs

```diff
@@ -227,7 +227,7 @@ impl ClusterConnection {
                 _ => panic!("No reach."),
             };
 
-            if let Ok(mut conn) = connect(
+            if let Ok(mut conn) = Self::connect(
```

Comment:
This needs to be part of the previous commit; each commit should compile and pass tests.

Reply:
I squash this commit with previous commit, I'll apply this rule to other PRs. Thanks!